server/services/utils/content.js

```diff
@@ -25,9 +25,9 @@ const removeCreatorAndUpdaterInfo = (item) => {
   delete item.updatedBy;
 }
 
-const findAll = async (uid) => {
+const findAll = async (uid, populateSchema) => {
   const result = await strapi.entityService.findMany(uid, {
-    populate: '*',
+    populate: populateSchema ? populateSchema : '*',
```

Comment:
Does that work with other entities now ?

Reply:
yes it does, the new changes only apply for the simulations for now, the rest are exported as normal